app/controllers/api/v0/plans_controller.rb

```diff
@@ -61,10 +61,44 @@ def index
     if params[:per_page].present? && params[:per_page].to_i > Rails.configuration.branding[:application][:api_max_page_size]
       params[:per_page] = Rails.configuration.branding[:application][:api_max_page_size]
     end
-    @plans = paginate @user.org.plans.includes( [ {answers: :question_options} ,
-      template: [ { phases: { sections: { questions: :question_format } } }, :org]
+    @plans = paginate @user.org.plans.includes( [ {roles: :user}, {answers: :question_options} ,
+      template: [ { phases: { sections: { questions: [:question_format, :themes ] } } }, :org]
       ])
+    
+    # Filter on list of users
+    user_ids = extract_param_list(params, "user")
+    @plans = @plans.where(roles: {user_id: user_ids}) if user_ids.present?
```

Comment:
do we want to include all roles here (e.g. commenter and reviewer)?

Reply:
Reviewer is no longer a role,  But we do probably want to keep this to editors/administrators/creators.  

I have added a helper-function to the role model to allow us to extract the numbers which correspond to a bit_value

Reply:
ah right. forgot we had removed the reviewer role. thanks for adding the helper method